app/controllers/api/v2x0.rb

```diff
@@ -0,0 +1,82 @@
+module Api
+  module V2x0
+    class RootController < ApplicationController
+      def openapi
+        render :json => ::Insights::API::Common::OpenApi::Docs.instance["2.0"].to_json
+      end
+    end
+
+    # Same behavior as api/v1.0/
+    class AvailabilitiesController              < Api::V1x0::AvailabilitiesController; end
```

Comment:
Do we want to inherit this from `Api::V1::Avail...` instead of `V1x0` ?  Theoretically if we released a `V1x1` we'd want the most recent in v2 right?

Reply:
`V1x0` inherits from `V1` and I think `V2x0` should inherit from the latest version 1 controllers which are currently `V1x0`.  In the future if we release a version 1.1, we should update the `V2x0` controllers to inherit from that.

Reply:
Would `Api::V1` _always_ be the latest in the `1.*` line though?  I'm thinking we wouldn't have to update anything in `V2x0` if we updated to a `V1x1`.

This is mostly a moot point since I doubt we'll ever update v1 if we're going to deprecate it, mostly trying to figure out the process going forward

Reply:
It's currently the other way around, but it would be nice to not have to update it.
```
module Api
  module V1x0
    class AvailabilitiesController              < Api::V1::AvailabilitiesController; end
    class ClustersController                    < Api::V1::ClustersController; end
    class ContainerGroupsController             < Api::V1::ContainerGroupsController; end
```
> This is mostly a moot point since I doubt we'll ever update v1 if we're going to deprecate it, mostly trying to figure out the process going forward

I agree, always good to review the process and make improvements along the way.